.percy.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 2,
+  "snapshot": {
+    "widths": [1280],
+    "minHeight": 1024,
+    "percyCSS": ".visual-test\\:invisible, .Toastify { visibility: hidden; }"
```

Comment:
So this is specifically to hide certain elements from Percy and more generally define any special CSS classes that Percy will apply?

Reply:
Yep visual testing needs stable snapshots across runs, and sometimes you've got dynamic elements on the page which you need to hide for Percy to achieve a stable snapshot. Which is where this "percyCSS" comes into play. For HTML which is under your control you can use this `visual-test:invisible` util style (the `.Toastify` is to target the toaster library)

Reply:
Ah got it. That makes a lot more sense now, thanks 👍🏻